app/pkg/email/mailgun/mailgun_test.go

```diff
@@ -198,3 +198,31 @@ func TestBatch_Success(t *testing.T) {
 	</body>
 </html>`)
 }
+
+func TestGetBaseURL(t *testing.T) {
+	// Memorize the original setting since we are about to change it
+	var initialRegion = env.Config.Email.Mailgun.Region
```

Comment:
You must just call `RegisterT(t)` at the beginning of the test like all other tests. If you try to run this test in isolation it'll fail with a message that you forgot to call RegisterT.

Another benefit of RegisterT is that it'll automatically restore `env.Config` object, so you don't need to manually memorize and restore it :)

Reply:
Ah, no wonder I did not notice any issues, I always ran at least all of the Mailgun tests, never just a single one. Good to know though, I'll fix it (: